### PR TITLE
Add ASP.NET Core web module template

### PR DESCRIPTION
**Pull request by GlebChili**

### CODE REVIEW

**Comment by nostripledev** [comment]
good branch

File: ProjectTemplates/gmodnet-module-web/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+TemplateModuleWeb.sln
```
Comment (SupinePandora43) [inline ProjectTemplates/gmodnet-module-web/.gitignore:1]: Why ignore solution file?

Comment (GlebChili) [reply ProjectTemplates/gmodnet-module-web/.gitignore:1]: In order to maintain compatibility with other .NET SDK templates, like `console`, `classlib`, etc.

File: global.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
   "sdk": {
-    "version": "5.0.302"
+    "version": "6.0.100-rc.2.21505.57"
```
Comment (SupinePandora43) [inline global.json:3]: Use released dotnet version.

**Review by SupinePandora43 — CHANGES_REQUESTED** [review]

File: global.json (hunk 2 above)
Comment (GlebChili) [reply global.json:3]: I will fix it in the next PR, where i will update main template.

**Review by SupinePandora43 — APPROVED** [review]

**Pull request merged**